dosomething-qa/fastly-frontend/takeovers/election.html

```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
```

Comment:
We should maybe just remove this (an old static HTML takeover from the election)! But didn't want to make any changes in this PR that'd result in an actual configuration change.

Reply:
Yeah... is there a ticket regarding removing it? If not we should defs add one!

Reply:
I need to make one! 📝

Reply:
Created [#170936349](https://www.pivotaltracker.com/story/show/170936349) for this.